### PR TITLE
add some config item for PageStorage V3

### PR DESCRIPTION
**Pull request by lidezhu**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -183,6 +183,12 @@ delta_index_cache_size = 0
     # TiFlash 存储引擎的压缩级别，默认为 1。如果 dt_compression_method 设置为 LZ4，推荐将该值设为 1；如果 dt_compression_method 设置为 zstd ，推荐将该值设为 -1 或 1，设置为 -1 的压缩率更小，但是读性能会更好；如果 dt_compression_method 设置为 LZ4HC，推荐将该值设为 9。
     dt_compression_level = 1
 
+    ## 从 v6.2.0 引入，代表 PageStorage V3 单个数据文件中有效数据的最低比例，当某些数据文件的有效数据比例低于该值时，会触发 GC 对相关文件的数据进行整理。默认为 0.5。
```
Comment (flowbehappy) [inline tiflash/tiflash-configuration.md:186]:
```suggestion
    ## 从 v6.2.0 引入，代表 PageStorage 单个数据文件中有效数据的最低比例，当某些数据文件的有效数据比例低于该值时，会触发 GC 对相关文件的数据进行整理。默认为 0.5。
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -183,6 +183,12 @@ delta_index_cache_size = 0
...
+    dt_storage_blob_heavy_gc_valid_rate = 0.5
+
+    ## 从 v6.2.0 引入，PageStorage V3 数据文件中的数据块会按照此参数对齐 IO 边界。默认值为 0，代表不进行 IO 边界对齐.
```
Comment (flowbehappy) [inline tiflash/tiflash-configuration.md:189]: Looks like this setting is not useful. I think we don't need to mention it here.

Comment (lidezhu) [reply tiflash/tiflash-configuration.md]: removed.

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -183,6 +183,12 @@ delta_index_cache_size = 0
     # TiFlash 存储引擎的压缩级别，默认为 1。如果 dt_compression_method 设置为 LZ4，推荐将该值设为 1；如果 dt_compression_method 设置为 zstd ，推荐将该值设为 -1 或 1，设置为 -1 的压缩率更小，但是读性能会更好；如果 dt_compression_method 设置为 LZ4HC，推荐将该值设为 9。
     dt_compression_level = 1
 
+    ## 从 v6.2.0 引入，代表 PageStorage 单个数据文件中有效数据的最低比例，当某些数据文件的有效数据比例低于该值时，会触发 GC 对相关文件的数据进行整理。默认为 0.5。
+    dt_storage_blob_heavy_gc_valid_rate = 0.5
+
+    ## 从 v6.2.0 引入，PageStorage 数据文件中的数据块会按照此参数对齐 IO 边界。默认值为 0，代表不进行 IO 边界对齐.
+    dt_storage_blob_block_alignment_bytes = 0
+
```
Comment (flowbehappy) [inline tiflash/tiflash-configuration.md:191]:
```suggestion
```
Remove this setting.

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -183,6 +183,12 @@ delta_index_cache_size = 0
     # TiFlash 存储引擎的压缩级别，默认为 1。如果 dt_compression_method 设置为 LZ4，推荐将该值设为 1；如果 dt_compression_method 设置为 zstd ，推荐将该值设为 -1 或 1，设置为 -1 的压缩率更小，但是读性能会更好；如果 dt_compression_method 设置为 LZ4HC，推荐将该值设为 9。
     dt_compression_level = 1
 
+    ## 从 v6.2.0 引入，代表 PageStorage 单个数据文件中有效数据的最低比例，当某些数据文件的有效数据比例低于该值时，会触发 GC 对相关文件的数据进行整理。默认为 0.5。
+    dt_storage_blob_heavy_gc_valid_rate = 0.5
```
Comment (flowbehappy) [inline tiflash/tiflash-configuration.md:187]:
```suggestion
    ## 从 v6.2.0 引入，代表 PageStorage 单个数据文件中有效数据的最低比例，当某些数据文件的有效数据比例低于该值时，会触发 GC 对相关文件的数据进行整理。默认为 0.5。
    dt_page_gc_threshold = 0.5
```

File: tiflash/tiflash-configuration.md [hunk 5]
```diff
@@ -183,6 +183,9 @@ delta_index_cache_size = 0
     # TiFlash 存储引擎的压缩级别，默认为 1。如果 dt_compression_method 设置为 LZ4，推荐将该值设为 1；如果 dt_compression_method 设置为 zstd ，推荐将该值设为 -1 或 1，设置为 -1 的压缩率更小，但是读性能会更好；如果 dt_compression_method 设置为 LZ4HC，推荐将该值设为 9。
     dt_compression_level = 1
 
+    ## 从 v6.2.0 引入，代表 PageStorage 单个数据文件中有效数据的最低比例，当某些数据文件的有效数据比例低于该值时，会触发 GC 对相关文件的数据进行整理。默认为 0.5。
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:186]:
```suggestion
    ## 从 v6.2.0 引入，表示 PageStorage 单个数据文件中有效数据的最低比例。当某个数据文件的有效数据比例低于该值时，会触发 GC 对该文件的数据进行整理。默认为 0.5。
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

**Review by flowbehappy — APPROVED** [review]

**Pull request merged**